README.md

```diff
@@ -23,38 +21,11 @@ for more information.
    requests for the CIRCT repository, and gain commit access using the [standard
    LLVM policies](https://llvm.discourse.group/c/Projects-that-want-to-become-official-LLVM-Projects/circt/).
 
-Also take a look at the following diagram, which gives a brief overview of the current [dialects and how they interact](docs/includes/img/dialects.svg):
-
-<p align="center"><img src="docs/includes/img/dialects.svg"/></p>
-
-## Motivation
```

Comment:
I'm not sure that dropping this adds a lot of value - a motivation section in the getting started guide is a bit weird.  @stephenneuendorffer WDYT about this proposal?

Reply:
Yeah, correct. The right place of motivation should be in the charter doc.

Reply:
Not sure we should update the charter doc once committed? Otherwise, I will just leave the motivation text in README.md.

Reply:
IIRC, the charter is (or perhaps intended to be) essentially a longer version of this motivation. I would keep the motivation in the README.